git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
* Not all projects have JIRA + GitHub integration. So forcing the case for that reason doesn't make sense. 
* Personally, I think those details should be in the PR and not in the commit message. Commits can be super small, even breaking changes, to a project. The PR is the atomic change. For any given commit you're only ever one click away from the PR that introduced it.